src/partials/partial-swagger.hbs

```diff
@@ -1,7 +1,12 @@
 <div swagger-ui
-  input-type="url"
+  input-type="{{ swagger.inputType }}"
+  {{# compare swagger.inputType '===' 'json' }}
+  input="{{ getDataByName swagger.input }}"
+  {{/ compare }}
+  {{# compare swagger.inputType '===' 'url' }}
```

Comment:
How many input types are there? Seems that this wouldn't work if I didn't include either `json` or `url` as the type? If the default is `url`, then perhaps we could do a catch-all helper here:

```
<div swagger-ui
  input-type="{{ swagger.inputType }}"
  {{# compare swagger.inputType '===' 'json' }}
    input="{{ getDataByName swagger.input }}"
  {{ else }}
    input="'{{ swagger.input }}'"
  {{/ compare }}
  {{# if swagger.template }}
    template-url="{{ swagger.template }}"
  {{/ if }}
></div>
```

If there ARE more options than `json` and `url`, what you have is okay; some folks' implementations just might not work if they add an incorrect `inputType` (which may be acceptable for this).